internal/frontend/cuefrontendopaque/tests.go

```diff
@@ -48,9 +51,20 @@ func parseTest(ctx context.Context, env *schema.Environment, pl parsing.EarlyPac
 		return nil, err
 	}
 
+	envVars, err := bits.Env.Parsed(pkg.Location.PackageName)
+	if err != nil {
+		return nil, err
+	}
+
 	out := &schema.Test{
 		Name:             name,
 		ServersUnderTest: bits.Servers,
+		Driver: &schema.Binary{
```

Comment:
It feels confusing to have a Driver which is not an actual binary. I suggest just having a schema.BinaryConfig here, which then gets merged as you're doing below.